test/run-drun/failures.log

```diff
@@ -0,0 +1,3 @@
+Wed Jun 21 10:08:57 MDT 2023 region0-stable-mem-custom-max.mo
+Wed Jun 21 10:09:28 MDT 2023 region0-stable-mem-max.mo
+Wed Jun 21 10:09:54 MDT 2023 stable-mem-custom-max.mo
```

Comment:
@crusso This file is up to date, with respect to my local run of the `test-feature.sh` script on this branch.